apps/desktop/src/lib/vbranches/types.ts

```diff
@@ -282,12 +283,8 @@ export class DetailedCommit {
 
 	get status(): CommitStatus {
 		if (this.isIntegrated) return 'integrated';
```

Comment:
We want integrated to take priority as a stats. The rest should be mutually exclusive